docs/quickstarts/packet.md

```diff
@@ -141,126 +124,112 @@ ssh-add ~/.ssh/id_rsa
 
 Success - cluster is healthy and nodes are ready!
 ```
 
-Use the generated `kubeconfig` file to access the Kubernetes cluster and list nodes.
+## Verification
+
+Use the generated `kubeconfig` file to access the cluster:
 
 ```console
-export KUBECONFIG=./lokomotive-assets/cluster-assets/auth/kubeconfig
+export KUBECONFIG=$(pwd)/assets/cluster-assets/auth/kubeconfig
 kubectl get nodes
 ```
 
+Sample output:
+
+```console
+NAME                            STATUS   ROLES    AGE   VERSION
+lokomotive-demo-controller-0      Ready    <none>   33m   v1.17.4
+lokomotive-demo-pool-1-worker-0   Ready    <none>   33m   v1.17.4
+lokomotive-demo-pool-1-worker-1   Ready    <none>   33m   v1.17.4
+```
+
 ## Using the cluster
 
-At this point you have access to the Kubernetes cluster and can use it!
-If you don't have Kubernetes experience you can check out the [Kubernetes
-Basics official
-documentation](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/)
-to learn about its usage.
+At this point you should have access to a Lokomotive cluster and can use it to deploy applications.
 
-**Note**: Lokomotive sets up a pretty restrictive Pod Security Policy that
-disallows running containers as root by default, check the [Pod Security Policy
-documentation](../concepts/securing-lokomotive-cluster.md#cluster-wide-pod-security-policy)
-for more details.
+If you don't have any Kubernetes experience, you can check out the [Kubernetes
+Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/) tutorial.
+
+>NOTE: Lokomotive uses a relatively restrictive Pod Security Policy by default. This policy
+>disallows running containers as root. Refer to the
+>[Pod Security Policy documentation](../concepts/securing-lokomotive-cluster.md#cluster-wide-pod-security-policy)
+>for more details.
 
 ## Cleanup
 
-To destroy the Lokomotive cluster, execute the following command:
+To destroy the cluster, execute the following command:
 
 ```console
 lokoctl cluster destroy --confirm
 ```
 
-You can safely delete the working directory created for this quickstart guide if you no longer
-require it.
+You can now safely delete the directory created for this guide if you no longer need it.
 
 ## Troubleshooting
 
-### Stuck at copy controller secrets
+In case the deployment fails, adding the `-v` flag to `lokoctl cluster apply` can provide useful
+information.
 
-If there is an execution error or no progress beyond the output provided below:
+### Stuck at "copy controller secrets"
 
 ```console
 ...
-module.packet-mycluster.null_resource.copy-controller-secrets: Still creating... (8m30s elapsed)
-module.packet-mycluster.null_resource.copy-controller-secrets: Still creating... (8m40s elapsed)
+module.packet-lokomotive-demo.null_resource.copy-controller-secrets: Still creating... (8m30s elapsed)
+module.packet-lokomotive-demo.null_resource.copy-controller-secrets: Still creating... (8m40s elapsed)
```

Comment:
You're right, that's an old output sample. Will fix.

I don't think we should tell the user to run with `-v`. If we think the Terraform output is important enough for a sample quickstart cluster, we should print this output by default.

Reply:
Scratch the above - this *is* the "Troubleshooting" section.

>In case the deployment fails, adding the `-v` flag to `lokoctl cluster apply` can provide useful
information.

Isn't this ^ clear enough? It appears right before the line you commented on.

Reply:
My point is that, to me, it is not clear that running with `-v` is required to see this output. I'd think that if I didn't see this, I might not have hit this issue. 

One way to improve it, IMHO, would be to say it clearer in the line just above this. Will comment there

Reply:
See my previous comments regarding `-v`. I suggest we include `-v` in the instructions`.

Reply:
Fully agreed! :)